SWLOR.Game.Server/Feature/QuestDefinition/ViscaraQuestDefinition.cs

```diff
@@ -542,5 +544,108 @@ private void JoiningTheRepublic()
                 .AddGoldReward(2000)
                 .AddItemReward("key_rep_01", 1);
         }
+
+        private void MedicalEquipmentForShelby()
+        {
+            _builder.Create("medical_equipget", "Medical Equipment for Shelby")
+
+                .AddState()
+                .SetStateJournalText("Nurse Shelby in Veles Medical Center needs ten new medical beds and the schematics for a new medical center.")
+                .AddCollectItemObjective("structure_0137", 10)
+                .AddCollectItemObjective("structure_5002", 1)
+
+                .AddState()
+                .SetStateJournalText("Well done, you've gathered what Shelby needed. Make sure you talk to her for a reward.")
+
+                .AddGoldReward(5000)
+                .AddXPReward(2500)
+                .AddItemReward("recipe_fabmedic1", 1);
+        }
+
+        private void SpiceOneSmallFavour()
+        {
+            _builder.Create("spice_onesmallfavour", "Spice: One Small Favour")
+
+                .AddState()
+                .SetStateJournalText("Stephen needs you to gather some different fiberplast so he can make some cool new rags he saw in a Magazine.")
+                .AddCollectItemObjective("fiberp_ruined", 12)
+                .AddCollectItemObjective("fiberp_flawed", 13)
+                .AddCollectItemObjective("fiberp_good", 21)
+                .AddCollectItemObjective("fiberp_imperfect", 19)
+
+                .AddState()
+                .SetStateJournalText("Fiberplast handed over - Looks like he'll get to wear those new rags now!")
+
+                .AddState()
+                .SetStateJournalText("Now Stephen needs you to get him some electronics so that he can make a sick new electric guitar.")
+                .AddCollectItemObjective("elec_ruined", 12)
+                .AddCollectItemObjective("elec_flawed", 19)
+                .AddCollectItemObjective("elec_good", 24)
+                .AddCollectItemObjective("elec_imperfect", 17)
+
+                .AddState()
+                .SetStateJournalText("You gave him the electronics, but not sure if he even has power down here.")
```

Comment:
Adjust for some clarity: "You gave him the electronics but you're not sure if he even has power down here."